cfme/storage/volume.py

```diff
@@ -163,10 +164,23 @@ def is_displayed(self):
             self.entities.title.text == expected_title and
             self.entities.breadcrumb.active_location == expected_title)
 
+    flash = FlashMessages('.//div[@id="flash_msg_div"]/div[@id="flash_text_div" or '
```

Comment:
This should be redundant, as it should be available through inheritence of VolumeView -> BaseLoggedInPage  (where flash is defined)